scripts/tests

```diff
@@ -0,0 +1 @@
+../tests/
```

Comment:
I wanted to use the fixtures defined in `ralph/tests` from `ralph/scripts/generate_docs.py` but wasn't able to import it from there.
So I created a symbolic link. I suppose this is bad practice, is there a better way ?